tests/unit/test_avro_merge.py

```diff
@@ -0,0 +1,83 @@
+"""
+karapace - Unit Test of AvroMerge class
+
+Copyright (c) 2023 Aiven Ltd
+See LICENSE for details
+"""
+
+
+from karapace.schema_models import AvroMerge
+from karapace.utils import json_decode, json_encode
+from unittest.mock import MagicMock
+
+import pytest
+
+
+class TestAvroMerge:
+    @pytest.fixture
+    def setup_avro_merge(self):
+        schema_str = '{"type": "record", "name": "Test", "fields": [{"name": "field1", "type": "string"}]}'
+        dependencies = {"dependency1": MagicMock(schema=MagicMock(schema_str='{"type": "string"}', dependencies=None))}
```

Comment:
Do you really need to mock the dependency in this simple case?

Reply:
It is a basic unit test for the AvroMerge functionality.